src/nextcloud/utilities/nextcloud-utilities

```diff
@@ -71,3 +71,57 @@ set_previous_cronjob_interval()
 {
 	snapctl set private.nextcloud.cron-interval="$1"
 }
+
+# Based on https://docs.nextcloud.com/server/21/admin_manual/installation/source_installation.html?highlight=htaccess%20rewritebase#pretty-urls
+is_htaccess_rewritebase_valid()
+{
+	local REWRITEBASE
```

Comment:
This isn't valid posix shell, looks like bash. That's why circle is failing-- this is using a posix shell.

Reply:
hm... I've removed that. But there seems to be something else that isn't compliant. Do you know what it is?

Reply:
I would resolve it myself but I cannot access the log that should be provided by circleci...

Reply:
This is the error that the static-tests CircleCI is reporting:
```
In /root/nextcloud-snap/tests/../src/nextcloud/utilities/nextcloud-utilities line 79:
	if ! [ "$REWRITEBASE" = '/' ] && ! [[ "$REWRITEBASE" =~ ^/[a-z0-9]+$ ]]; then
                                           ^-- SC2039: In POSIX sh, [[ ]] is undefined.
```

Reply:
thanks! I'll have a look how to fix that.

Reply:
I think I fixed it with c33ae03

Reply:
Seems like this didn't work. What is now the problem?

Reply:
Now the static-tests have passed, so the problem is with the dynamic tests; all of them are failing, I guess because `httpd` is not starting properly:
```
Change operating mode
  production (FAILED - 1)
  debug (FAILED - 2)
  invalid
  
An error occurred in an `after(:context)` hook.
Failure/Error: fail "Timed out trying to access Nextcloud: #{uri.to_s}"

RuntimeError:
  Timed out trying to access Nextcloud: http://localhost
# ./spec/spec_helper.rb:219:in `rescue in wait_for_nextcloud'
# ./spec/spec_helper.rb:201:in `wait_for_nextcloud'
# ./spec/change_mode_spec.rb:6:in `block (2 levels) in <top (required)>'
# ------------------
# --- Caused by: ---
# Timeout::Error:
#   execution expired
#   ./spec/spec_helper.rb:215:in `sleep'

Change PHP memory limit
  unlimited (FAILED - 3)
  bytes (FAILED - 4)
  invalid (FAILED - 5)
  
An error occurred in an `after(:context)` hook.
Failure/Error: fail "Timed out trying to access Nextcloud: #{uri.to_s}"

RuntimeError:
  Timed out trying to access Nextcloud: http://localhost
# ./spec/spec_helper.rb:219:in `rescue in wait_for_nextcloud'
# ./spec/spec_helper.rb:201:in `wait_for_nextcloud'
# ./spec/change_php_memory_limit_spec.rb:4:in `block (2 levels) in <top (required)>'
# ------------------
# --- Caused by: ---
# Timeout::Error:
#   execution expired
#   ./spec/spec_helper.rb:215:in `sleep'

Change ports
  http
    http (FAILED - 6)
    
An error occurred in an `after(:context)` hook.
Failure/Error: fail "Timed out trying to access Nextcloud: #{uri.to_s}"

RuntimeError:
  Timed out trying to access Nextcloud: http://localhost
# ./spec/spec_helper.rb:219:in `rescue in wait_for_nextcloud'
# ./spec/spec_helper.rb:201:in `wait_for_nextcloud'
# ./spec/change_port_spec.rb:5:in `block (3 levels) in <top (required)>'
# ------------------
# --- Caused by: ---
# Timeout::Error:
#   execution expired
#   ./spec/spec_helper.rb:215:in `sleep'
  https
    https (FAILED - 7)
    http still redirects to unchanged https (FAILED - 8)
    http redirects to changed https (FAILED - 9)
    Let's Encrypt challenge request (FAILED - 10)
...
```

Reply:
I think I've figured out what the problem was. Should be fixed with da2814e